test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/MutableObjectModelBinderTest.cs

```diff
@@ -942,53 +941,15 @@ public void ProcessDto_ValueTypePropertyWithBindRequired_SetToNull_CapturesExcep
         }
 
         [Fact]
-        [ReplaceCulture]
-        public void ProcessDto_MissingDataForRequiredFields_NoErrors()
```

Comment:
Deleted this one because it was just redundant. Retrofitted other test to cover important scenarios:
- value type with no value -> error
- value type with optional with no value -> no error
- nullable value type with no value -> no error